example-config/etesync.ini

```diff
@@ -0,0 +1,18 @@
+# uwsgi configuration file
+# typical location of this file would be /etc/uwsgi/sites/etesync.ini
+
+[uwsgi]
+folder = /path/to/etesync
+
+socket = /run/uwsgi/etesync.sock
+chown-socket = EtesyncUser:www-data
+chmod-socket = 660
+vacuum = true
+
+
+uid = etes
+chdir = %(folder)
+home = %(folder)/.venv
+module = etesync_server.wsgi:application
```

Comment:
I don't know what it should be, but I only have etesync_server.wsgi, no :application